From b670d634789822715e21c0f567b39f1e34cd0aef Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Mon, 8 Jun 2015 23:20:24 +0800
Subject: [PATCH 0779/1221] MLK-11075: arm: dts: enable usb otg1 and otg2 port
 for imx7d-19x19-lpddr3-arm2 board

Enable otg1 and otg2 on imx7d-19x19-lpddr3-arm2 board with dr mode.

Signed-off-by: Li Jun <jun.li@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts |   10 ++++++++++
 1 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
index f74b6d8..9221272 100644
--- a/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
@@ -153,6 +153,16 @@
 	};
 };
 
+&usbotg1 {
+	vbus-supply = <&reg_usb_otg1_vbus>;
+	status = "okay";
+};
+
+&usbotg2 {
+	vbus-supply = <&reg_usb_otg2_vbus>;
+	status = "okay";
+};
+
 &usdhc1 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc1>;
-- 
1.7.5.4

